integrated_channels/integrated_channel/management/commands/transmit_content_metadata.py

```diff
@@ -39,6 +39,10 @@ def handle(self, *args, **options):
         """
         username = options['catalog_user']
 
+        # Since Cornerstone is following pull content model we don't need to include CSOD customers in a metadata
+        # transmission
```

Comment:
this comment doesn't really make sense in the context of the code it's above.
Like all that code is is adding the option, so talking about cornerstone seems irrelevant here. I'm not sure we really need a comment at all.

Reply:
I had asked Alex to add a comment without realizing the code already had a clear model for showing which options were available for which LMSs. I agree comment can be removed.

Reply:
agreed